components/core/src/os/process/exec/unix.rs

```diff
@@ -37,6 +40,14 @@ pub fn hook_command<X, I, K, V>(executable: X, env: I, ids: Option<(Uid, Gid)>)
         with_user_and_group_information(&mut cmd, uid, gid);
     }
 
+    unsafe {
```

Comment:
I think I would add a comment here about why we had to do this with a link to the rust github pr.

Reply:
Added comment per feedback and will now merge.